src/SourceBuild/content/repo-projects/package-source-build.proj

```diff
@@ -9,61 +9,58 @@
 
   <ItemGroup>
     <RepositoryReference Include="installer" />
+    <RepositoryReference Include="scenario-tests" />
```

Comment:
Interesting. Why does package-source-build depend on scenario-tests?

Reply:
Fixed in https://github.com/dotnet/installer/pull/18557/commits/308159fcda50b3fb5ae529ea2c96bcedf63aecc7